lib/bindings/python/tests/test_canary_health_race.py

```
@@ -0,0 +1,175 @@
+    """
+    await asyncio.sleep(999)
+    yield {"error": "should never reach here"}
+
+
+def _poll_health(port, path="/health", timeout=2):
+    """Poll health endpoint, return (status_code, body_dict) or None on connection error."""
+    try:
+        resp = requests.get(f"http://localhost:{port}{path}", timeout=timeout)
+        try:
+            body = resp.json()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)